### ISSUE TITLE
"--mirror" ignored when using "single backup"

### ISSUE THREAD

**Issue by stefangries** [joeyates/imap-backup#194]
Hello everyone,

When using the following command, the mirror flag is ignored:

```
docker run --rm -v /hostpath/data:/data ghcr.io/joeyates/imap-backup:latest \
  imap-backup single backup \
  --mirror --email XX@XX.XX --password XXXX --server mail.XX.XX \
  --path /data/XX
```

That results in mails, that are removed from the server, are not removed from the local mbox files. Is this a bug or am I doing something wrong?

**Comment by stefangries** [comment]
Thank you very much for your super-fast reply and clarifying this! As the cli takes this argument (https://github.com/joeyates/imap-backup/blob/9f1d1d96ccdc75e5afb1941e63b44228c27f840d/lib/imap/backup/cli/single/backup.rb#L31), I thought this is already useable.

Is there a way to do IMAP to local MBOX backups with the current feature set where mirror is supported?

**Comment by joeyates** [comment]
@stefangries You can create a config file and use that with Docker/Podman.

Something like this:

```sh
docker \
  run \
  --rm \
  -v /hostpath/data:/data \
  -v /hostpath/config:/config \
  ghcr.io/joeyates/imap-backup:latest \
  imap-backup backup \
  --config /config/my-config.json
```

**First**, you need to create `/hostpath/config/my-config.json`.

You can do that with Docker/Podman too:

```sh
docker \
  run \
  --rm \
  -ti \
  -v /hostpath/data:/data \
  -v /hostpath/config:/config \
  ghcr.io/joeyates/imap-backup:latest \
  imap-backup setup \
  --config /config/my-config.json
```

(Remember the `-ti`!)

**Comment by stefangries** [comment]
Thank you! Is `-ti` really necessary? I'm using Synology Task Scheduler which has no tty and gives me `the input device is not a TTY `error when using `-ti`.

**Comment by joeyates** [comment]
Fixed in [v14.6.1](https://github.com/joeyates/imap-backup/releases/tag/v14.6.1)
